api/agent/agent_test.go

```diff
@@ -49,7 +49,7 @@ func TestCallConfigurationRequest(t *testing.T) {
 		}, nil,
 	)
 
-	a := New(ds, ds, new(mqs.Mock), AgentTypeFull)
+	a := New(NewCachedDataAccess(NewDirectDataAccess(ds, ds, new(mqs.Mock))))
```

Comment:
for tests we probably don't want the cache, since we may want to e.g. update a route, make sure it doesn't work after update. seems this doesn't break anything for the time being though